pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -196,7 +196,7 @@ in stdenv.mkDerivation {
   '' + ''
 
     # libredirect causes chromium to deadlock on startup
-    export LD_PRELOAD="\$(echo -n "\$LD_PRELOAD" | tr ':' '\n' | ${gnugrep}/bin/grep -v /lib/libredirect\\\\.so$ | tr '\n' ':')"
+    export LD_PRELOAD="\$(echo -n "\$LD_PRELOAD" | ${coreutils}/bin/tr ':' '\n' | ${gnugrep}/bin/grep -v /lib/libredirect\\\\.so$ | ${coreutils}/bin/tr '\n' ':')"
```

Comment:
Please always comment _why_. You already wrote it into the PR :)

Reply:
Do you mean in the Nix expression? I don't think that would be a good idea as the reason is pretty obvious and it would only make everything more verbose.

Reply:
Sorry, misunderstanding on my side.

I had somehow understood that there's a script in Chromium that requires `tr`, but the only place where `tr` is required is this direct invocation here.

This change makes total sense.

Reply:
ok, thanks